docs/kratos/emails-sms/01_sending-emails-smtp.mdx

```diff
@@ -5,41 +5,71 @@ sidebar_label: SMTP server configuration
 toc_max_heading_level: 4
 ---
 
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import CodeBlock from "@theme/CodeBlock"
+```
+
 The Ory Network comes with SMTP email sending configured out of the box. Ory emails are sent from this address:
 
 ```
 {project.name} via Ory <no-reply@courier-eu.mg.oryapis.com>
 ```
 
-## Send emails using your SMTP server
+:::info
 
-To use your own SMTP server for sending emails, adjust the SMTP configuration key directly in the Ory Identities (Kratos) config:
+You must send emails using your SMTP server to change the sender address (`from_address`) and sender name (`from_name`).
```

Comment:
Ah, missed this in my initial skim through. Thanks @tomekpapiernik , this is probably a good place, though it could use some explanation. I'll see if I can find anything on that.